snippets/code/precompiles/ecrecover.md

```diff
@@ -2,7 +2,7 @@
 const Web3 = require('web3');
 
 // Provider
-const web3 = new Web3('{{ networks.moonbase.rpc_url }}');
+const web3 = new Web3('RPC-API-ENDPOINT-HERE');
```

Comment:
I would use the public RPC endpoint of Moonbase Alpha for the examples. Not sure what you think

Reply:
these examples are for Moonbeam... should I update the tutorial so it uses Moonbase Alpha instead?

Reply:
it should have been for Moonbase Alpha the whole time, not sure what happened here but fixed it